logdna/provider.go

```diff
@@ -7,10 +7,10 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-type config struct {
-	ServiceKey string
-	URL        string
-	HTTPClient *http.Client
+type providerConfig struct {
+	serviceKey string
+	Host       string
```

Comment:
this isn't just the host, is it? looks like it will contain the protocol too

Reply:
`baseURL` is a good suggestion from your other comment.  Will do.